examples/getting-started-movielens/02-ETL-with-NVTabular.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Not important but we can remove <code>,</code> after <code>see</code>.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
comma removed